buildkite/bazelci.py

```diff
@@ -2998,19 +3001,15 @@ def fetch_incompatible_flags():
 def print_bazel_downstream_pipeline(
     task_configs, http_config, file_config, test_incompatible_flags, test_disabled_projects, notify
 ):
-    if not task_configs:
-        raise BuildkiteException("Bazel downstream pipeline configuration is empty.")
-
-    pipeline_steps = []
-    task_configs = filter_tasks_that_should_be_skipped(task_configs, pipeline_steps)
```

Comment:
task_configs here must be Bazel's task configs, no need to filter anything, also `pipeline_steps` is reset to empty afterwards.